### ISSUE TITLE
Allow role="none" on img with alt=""

### ISSUE THREAD

**Issue by dstorey** [w3c/html#812]
Currently the spec says only presentation should be allowed. I'm not sure if the spec isn't referencing Aria 1.1 yet, but it is in Cand Rec.

http://w3c.github.io/html/semantics-embedded-content.html#elementdef-img

**Comment by dstorey** [comment]
Validator bug at https://github.com/validator/validator/issues/472
